tests/integration/targets/vyos_ospf_interfaces/tests/cli/replaced.yaml

```diff
@@ -0,0 +1,60 @@
+---
+- debug:
+    msg: START vyos_ospf_interfaces replaced integration tests on connection={{
+      ansible_connection }}
+
+- include_tasks: _remove_config.yaml
+
+- include_tasks: _populate.yaml
+
+- block:
+
+    - name: Replace the existing configuration with the provided running configuration
+      register: result
+      vyos.vyos.vyos_ospf_interfaces: &id001
+        config:
+          - name: "eth0"
+            address_family:
+              - afi: "ipv4"
+                transmit_delay: 50
+                priority: 26
+                network: "point-to-point"
+              - afi: "ipv6"
+                dead_interval: 39
+          - name: "bond2"
+            address_family:
+              - afi: "ipv4"
+                transmit_delay: 45
+                bandwidth: 70
+                authentication:
+                  md5_key:
+                    key_id: 10
+                    key: "1111111111232345"
+              - afi: "ipv6"
+                passive: true
+        state: replaced
+
+    - become: true
+      vyos.vyos.vyos_facts:
+        gather_network_resources: ospf_interfaces
+
+    - assert:
+        that:
+          - result.commands|length == 8
+          - result.changed == true
+          - result.commands|symmetric_difference(replaced.commands) == []
+          - result.after|symmetric_difference(ansible_facts['network_resources']['ospf_interfaces']) == []
```

Comment:
Asserting before/after from gathered facts is not much useful, imo. Both are generated from the same code path.